File: html/Khushbu/C/4.c
#include<stdio.h>

int main(){

    // int age =17;

    // if(age>=18){
    //     printf("You can Vote\n");
    // }else{
    //     printf("You can Not Vote\n");
    // }




    int a=10;
    int b=13;
    int c=70;

    if(a>b && a>c){
        printf("%d is grater than %d and %d\n",a,b,c);
    }else if(b>a && b>c){
        printf("%d is grater than %d and %d\n",b,a,c);
    }else if(c>a && c>b){
        printf("%d is grater than %d and %d\n",c,a,b);
    }else{
        printf("All Numbers are Equal");
    }

    

    return 0;
}

File: html/Shivam1/php/4.php
<?php
$age=2;


// if($age>=18){
//     echo 'Your can vote ';
// }
// else{
//     echo "Your Can Not Vote";
// }


// echo ($age>18) ?  "You Can Vote" :  "You can not vote";


$a=19;
$b=10;
$c=10;

if($a>$b && $a>$c){
    echo $a . " is Grater Than " . $b . " and " .$c;
}
else if($b>$a && $b>$c){
    echo $b . " is Grater Than " . $a . " and " .$c;
}
else if($c>$a && $c>$b){
    echo $c . " is Grater Than " . $a . " and " .$b;
}
else{
    echo "All numbers are Equal";
}

?>



